news-explorer-frontend/src/utils/auth.js

```diff
@@ -0,0 +1,48 @@
+export const BASE_URL = 'https://api.explorer.students.nomoreparties.sbs';
+// export const BASE_URL = 'http://localhost:3000';
+
+const _checkResponse = (res) => {
```

Comment:
`_checkResponse` 

no need to duplicate `_checkResponse` 3 times in each file. You need to define it once and import to the file you need